### ISSUE TITLE
Ability to set per dashboard default timerange

### ISSUE THREAD

**Issue by PrFalken** [urbanairship/tessera#422]
Wouldn't it be cool ?
People here are asking for it.
It could be a new option in the dashboard's metadata. Or a per-dashboard cookie ?

**Comment by clohtd** [comment]
Yeah that would be nice.  I've actually setup browser shortcuts to accomplish this.
